src/renderers/__tests__/ReactCompositeComponentState-test.js

```diff
@@ -226,13 +211,12 @@ describe('ReactCompositeComponent-state', () => {
       ['forceUpdate', 'blue'],
       // unmountComponent()
       // state is available within `componentWillUnmount()`
-      ['componentWillUnmount', 'blue'],
+      ['componentWillUnmount', 'blue']
```

Comment:
We don't do this anywhere else in the codebase though. I'll need to take a minute to process this before I come to terms with it. :P